docs/en_us/platform_api/source/enrollment/overview.rst

```diff
@@ -0,0 +1,38 @@
+.. _edX Enrollment API Overview:
+
+################################################
+edX Enrollment API Overview
+################################################
+
+The edX Platform Enrollment API enables you to view user and course enrollment
```

Comment:
Suggestion: "Use the edX Platform Enrollment API to view user and course enrollment..."